driver/event_stats.h

```diff
@@ -8,4 +8,4 @@
 #define UNKNOWN_EVENTS_NUM 20
 
 /* This number should be increased when we add/remove a generic syscall */
-#define GENERIC_SYSCALLS_NUM 235
+#define GENERIC_SYSCALLS_NUM 246
```

Comment:
I will make a follow up PR to avoid defining this at compile time and instead loading it at runtime.